src/omv/common/sensor.h

```diff
@@ -137,6 +137,8 @@ typedef enum {
     IOCTL_GET_READOUT_WINDOW,
     IOCTL_SET_TRIGGERED_MODE,
     IOCTL_GET_TRIGGERED_MODE,
+    IOCTL_SET_FOV_PREFERRED,
+    IOCTL_GET_FOV_PREFERRED,
```

Comment:
Can you rename this to set/get WIDE_FOV please ?

Reply:
Or FOV_WIDE.

Reply:
Done